inc/theme-hooks.php

```diff
@@ -125,3 +125,123 @@ function alcatraz_output_footer_bottom() {
 		);
 	}
 }
+
+
+add_filter( 'alcatraz_primary_sidebar', 'alcatraz_sidebar_nav_menu', 5);
```

Comment:
This should be hooked on `alcatraz_before_primary_sidebar`. See `alcatraz_output_primary_sidebar` for where this hook is defined.